src/streamlink/plugins/pluzz.py

```diff
@@ -160,7 +176,21 @@ def _get_streams(self):
                     # Fallback bitrate (seems all France Televisions MP4 videos
                     # seem have such bitrate)
                     bitrate = '1500k'
-                yield bitrate, HTTPStream(self.session, video_url)
+                streams.append((bitrate, HTTPStream(self.session, video_url)))
+
+        if self.get_option("mux_subtitles") and videos['subtitles'] != []:
+            substreams = {}
+            for subtitle in videos['subtitles']:
+                # TTML subtitles are available but not supported by FFmpeg
+                if subtitle['format'] == 'ttml':
+                    continue
+                substreams[subtitle['type']] = HTTPStream(self.session, subtitle['url'])
```

Comment:
What's in `subtitle['type']`? `MuxedStream` will set whatever the key for the `subtitles` dict to the language of the subtitle.